examples/fastify-http-server-ts/package.json

```
@@ -0,0 +1,17 @@
+{
+  "name": "fastify-server",
```

Let it be the same as folder name.

```suggestion
  "name": "fastify-http-server-ts",
```